test/test.js

```diff
@@ -0,0 +1,118 @@
+
+import assert from "assert";
+import fs from "fs";
+import path from 'path';
+import { checkASThashes, checkCodeGen, checktestASTExists } from '../built/tests.js'
+import mkdirs from '../bin/mkdirs.mjs';
+import zappify from '../built/index.js'; 
+var files = fs.readdirSync('./test/contracts/')
+let chosenFile = files[Math.floor(Math.random() * files.length)]
+const inputFilePath = './test/contracts/'+chosenFile;
+const modifyAST = 'n';
+const inputFileName = path.parse(inputFilePath).name;
+// commander converts 'zapp-name' to 'zappName'
+const zappName = inputFileName;
```

Comment:
I like the random zapp! Could we also add a line that logs which zapp is being tested? Something like `logger.info('Testing ${zappName}')`

Reply:
Sure . I will log few more bits of information.